tests/jerry/date-tostring.js

```diff
@@ -32,6 +32,10 @@ assert (new Date (NaN).toDateString () == "Invalid Date");
 assert (new Date ("2015-02-13").toDateString () == "2015-02-13");
 assert (new Date ("2015-07-08T11:29:05.023").toDateString () == "2015-07-08");
 
+assert (new Date (NaN).toISOString () == "Invalid Date");
+assert (new Date ("2015-07-16").toISOString () == "2015-07-16T00:00:00.000Z");
+assert (new Date ("2015-07-16T11:29:05.023").toISOString () == "2015-07-16T11:29:05.023Z");
```

Comment:
Please move the new test cases to the end of the file and add test for TypeError too.